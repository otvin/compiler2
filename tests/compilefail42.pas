program compilefail42(output);
{illegal assignment to the control variable of a for statement}

var i:integer;

begin

	for i := 1 to 7 do
		begin
			writeln(i);
			i := i + 2;
		end;

end.
