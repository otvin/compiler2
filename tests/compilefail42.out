[1mtests/compilefail42.pas:11:13: [0m[1m[91merror:[0m Cannot assign a value to the control variable 'i' of a 'for' statement
    11 | 			i := i + 2;